src/main/java/ca/carleton/controllers/AccountController.java

```diff
@@ -16,6 +17,7 @@ public String loginForm(Model model){
 
     @PostMapping("/")
     public String loginSubmit(Model model, @ModelAttribute Customer customer){
+        this.customer = customer;
```

Comment:
This is gonna break if multiple people are using the website at the same time.

I think for now (since we don't have session cookies), just add a username parameter to the `requestData` mapping and make the js request button use the current username with Thymeleaf templating.